docs/content/integrations/embedded-elt.mdx

```diff
@@ -0,0 +1,126 @@
+---
+title: "Dagster Embedded ELT"
+description: Lightweight ELT with Dagster
+---
+
+# Dagster Embedded ELT
+
+This package provides a framework for building ELT pipelines with Dagster through helpful pre-built assets and resources. It is currently in experimental development, and we'd love to hear your feedback.
+
+This package currently includes a single implementation using <a href="https://slingdata.io">Sling</a> provides a simple way to sync data between databases and file systems.
+
+We plan on adding additional embedded ELT tool integrations in the future.
+
+## Relevant APIs
+
+| Name                                                                        | Description                                                                    |
+| --------------------------------------------------------------------------- | ------------------------------------------------------------------------------ |
+| <PyObject module="dagster_embedded_elt.sling" object="build_sling_asset" /> | The core Sling asset factory for building syncs                                |
+| <PyObject module="dagster_embedded_elt.sling" object="SlingResource" />     | The Sling Resource used for handing credentials to databases and object stores |
+
+---
+
+## Getting Started
+
+To get started with `dagster-embedded-elt` and Sling, first familiarize yourself with <a href="https://docs.slingdata.io/sling-cli/running-tasks">Sling's documentation</a>
```

Comment:
What about the docs should they be familiarizing themselves with? This might be nitpicky, but I think it's more _"Familiarize yourself with Sling by reading their docs"_ than _"Familiarize yourself with Sling's docs"_

This sentence is also missing ending punctuation.